nipype/interfaces/afni/utils.py

```diff
@@ -234,7 +234,7 @@ def aggregate_outputs(self, runtime=None, needed_outputs=None):
             m = re.search(pattern, line)
             if m:
                 d = m.groupdict()
-                outputs.trait_set(**{k: int(d[k]) for k in d.keys()})
+                outputs.trait_set(**{k: int(d[k]) for k in d})
```

Comment:
Here I think it would be better to iterate over items:

```suggestion
                outputs.trait_set(**{k: int(v) for k, v in d.items()})
```